### PR TITLE
Add exchange & chain clients

### PR DESCRIPTION
**Pull request by achilleas-kal**

### CODE REVIEW

File: examples/chain/21_MsgRevoke.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+func main() {
+	network := common.LoadNetwork("testnet", "k8s")
+	tmRPC, err := rpchttp.New(network.TmEndpoint, "/websocket")
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	senderAddress, cosmosKeyring, err := chainclient.InitCosmosKeyring(
+		"/Users/akalantzis/.injectived",
```
Comment (albertchon) [inline examples/chain/21_MsgRevoke.go:20]: we shouldn't commit this of course @achilleas-kal

Comment (albertchon) [reply examples/chain/21_MsgRevoke.go:20]: At minimum $HOME/.injectived but ideally also user configurable, with default being set to $HOME/.injectived if unconfigured

Comment (achilleas-kal) [reply examples/chain/21_MsgRevoke.go:20]: It's a dummy keyring/private key like we have in sdk-python, I think it makes the examples more clear @albertchon

Comment (achilleas-kal) [reply examples/chain/21_MsgRevoke.go:20]: Sure I'll change to $HOME/.injectived

**Review by nhannamsiu — APPROVED** [review]

**Comment by albertchon** [comment]
@achilleas-kal I see a lot of alerts from LGTM 
image
I think it's appropriate to panic(err) if err != nil in the keyring initialization, as the code shouldn't proceed if there's an issue with the keyring

**Comment by achilleas-kal** [comment]
Yeah makes sense, will fix @albertchon

**Pull request merged**